docs/data/sdks/android-kotlin/ampli.md

```diff
@@ -3,16 +3,17 @@ title: Android Ampli Wrapper
 description: Learn how to install and use the Amplitude Data Ampli Wrapper for the Android Java and Kotlin runtimes.
 ---
 
-!!!note
-    This page covers the Android Java and Kotlin runtimes. All (Itly) runtimes are deprecated. If you are still using an (Itly) runtime, see the **[migration guide](#migrating-from-an-itly-android-runtime)** to upgrade to the newest runtime. Docs for the Itly version are available **[here](../../deprecated-sdks/android.md)**.
 
 Amplitude Data supports tracking analytics events from Android apps written in Kotlin and Java.
 
 In Kotlin and Java, the tracking library exposes a type-safe function for every event in your team’s tracking plan. The function’s arguments correspond to the event’s properties and are strongly typed to allow for code completion and compile-time checks.
 
-!!!tip
+!!!info "Ampli Resources"
+    - [Android Java Ampli Examples](https://github.com/amplitude/ampli-examples/tree/main/android/java/v2/AmpliApp)
+    - [Android Kotlin Ampli Examples](https://github.com/amplitude/ampli-examples/tree/main/android/kotlin/v2/AmpliApp)
 
-    See example apps that use the Android Java and Kotlin runtimes on [GitHub](https://github.com/amplitude/ampli-examples/tree/main/android).
+!!!deprecated "Deprecated Itly runtime"
```

Comment:
I think this deprecation warning is confusing/aggressive. Most customers will not be coming from an Itly runtime, and this warning makes it look like the SDK on the current page is deprecated vs Itly. Can we make a difference admon for this? Or maybe revert it to a Note? cc @caseyamp 
<img width="746" alt="image" src="https://user-images.githubusercontent.com/5790872/187249917-935de200-f42a-45f2-9ba0-e6a4e64e72a2.png">

Reply:
I honestly have always thought the Itly deprecation notices aren't necessary -- like you said, most aren't coming from an Itly runtime so it's weird to even mention it on a non-itly SDK. If they're using Itly, why are they on this page? We should have these notices only on the deprecated runtime docs, IMO. @justin-fiedler

Reply:
The deprecated runtime docs don't have nav bar entry. I think these notices are the only entries for deprecated runtime docs now. Changing back to !!!note is better for now @justin-fiedler @caseyamp

Reply:
I'm okay with that option @bohan-amplitude.

Reply:
I think a future improvement could be to make a dedicated section for Itly migration and then remove mentions to Itly in the current SDKs. For now I think a note is ok.